requirements/edx/custom.txt

```diff
@@ -13,7 +13,7 @@
 git+https://github.com/edx-solutions/xblock-group-project.git@0.1.3#egg=xblock-group-project==0.1.3
 -e git+https://github.com/edx-solutions/xblock-adventure.git@0.2#egg=xblock-adventure==0.2
 -e git+https://github.com/open-craft/xblock-poll.git@v1.9.10#egg=xblock-poll==1.9.10
--e git+https://github.com/open-craft/problem-builder.git@3.5.1#egg=xblock-problem-builder==3.5.1
+-e git+https://github.com/open-craft/problem-builder.git@v3.5.2#egg=xblock-problem-builder==3.5.2
```

Comment:
FYI you're adding a `v` in here

Reply:
@moeez96 Yeah, previous tag of problem builder was created mistakenly without Semantic Versionv. Now open craft again created 3.5.2 with semantic version. so that' why pined the version with v.